Create gcloud-init.md (#11626)
tldr-pages/tldr: pages/common/gcloud-init.md

# gcloud init

> > This command launches an interactive workflow to setup gcloud or reinitialize configurations.

Reviewer: ```suggestion
> Launch an interactive workflow to setup `gcloud` or reinitialize configurations.
```

> More information: <https://cloud.google.com/sdk/gcloud/reference/init>.

> - Launch a Getting Started Workflow:

Reviewer 2: ```suggestion
- Launch a "Getting Started" workflow:
```

`gcloud init`

- Launch a workflow without diagnostics:

`gcloud init --skip-diagnostics`

> - Allow to use console for authentication:

Reviewer 2: ```suggestion
- Allow using the console for authentication:
```

> - Allow using the console for authentication:

Reviewer: ```suggestion
- Use the console for authentication:
```

`gcloud init --console-only`

> - Do not launch a browser:

Reviewer: https://cloud.google.com/sdk/gcloud/reference/init
> Prevent the command from launching a browser for authorization.
```suggestion
- Do not launch a browser for authorization:
```

> - Alternatively,prevent from launching a browser:

Reviewer 2: ```suggestion
- Do not launch a browser:
```

> 
> - Do not launch a browser:
> 
> `gcloud init --no-launch-browser`

Reviewer: This does exactly the same thing as `--console-only`.
https://cloud.google.com/sdk/gcloud/reference/init
> --console-only, --no-launch-browser
    Prevent the command from launching a browser for authorization. Use this flag if you are on a machine that does not have a browser and you cannot install the gcloud CLI on another machine with a browser. 
```suggestion
```
